### PR TITLE
Symfony 6 support, #3

### PR DESCRIPTION
**Pull request by tacman**
I made some of the easy changes for Symfony 6 support, mostly fixing the method signatures to include typehints, for #3

### CODE REVIEW

**Comment by pietereggink** [comment]
@theofidry could you please review this PR?

File: fixtures/HttpKernel/DummyKernel.php [hunk 1]
```diff
@@ -42,15 +45,15 @@ public function unserialize($serialized)
     /**
      * {@inheritdoc}
      */
-    public function handle(Request $request, $type = self::MASTER_REQUEST, $catch = true)
```
Comment (dunglas) [inline fixtures/HttpKernel/DummyKernel.php:45]: This constant isn't available before Symfony 5.3. Either bump the minimum version or hardcode the int `1`.

**Review by dunglas — CHANGES_REQUESTED** [review]

**Comment by FabienPapet** [comment]
Hello, any updates on this ? I can help if needed.

File: fixtures/HttpKernel/DummyKernel.php (hunk 1 above)
Comment (theofidry) [reply fixtures/HttpKernel/DummyKernel.php:45]: Will bump it in another PR

**Comment by theofidry** [comment]
Thank you @tacman

**Pull request merged**